src/routes/Library/Placeholder/Placeholder.less

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2017-2025 Smart code 203358507
+
+@import (reference) '~stremio/common/screen-sizes.less';
+
+.placeholder {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+    overflow-y: auto;
+
+    .title {
+        display: flex;
+        flex: 1;
+        align-items: flex-end;
+        font-size: 1.75rem;
+        font-weight: 400;
+        text-align: center;
+        color: var(--primary-foreground-color);
+        margin-bottom: 1rem;
+        opacity: 0.5;
+    }
+
+    .image-container {
+        display: flex;
+        flex: 1;
```

Comment:
```suggestion
        flex: none;
```

Reply:
Otherwise it's too spaced out

![image](https://github.com/user-attachments/assets/b7ce5ac1-d264-44cf-b8ff-be6613c745e9)